### PR TITLE
Makefile: Update build system to support setting QT_ACCOUNT_* variables

### PR DESCRIPTION
**Pull request by jcfr**

### CODE REVIEW

**Comment by jcfr** [comment]
@sjh26  This should allow you to move forward with updating the Dockerfile associated with `qt5-centos7`

**Comment by sjh26** [comment]
Thanks!

File: Docker/qt5-centos7/Dockerfile [hunk 1]
```diff
@@ -42,6 +42,7 @@ RUN \
     mesa-libGLU-devel \
     xcb-util-keysyms \
     xcb-util-image \
+    libxkbcommon-x11 \
```
Comment (jcfr) [inline Docker/qt5-centos7/Dockerfile:45]: Nitpick: Sort alphabetically

**Comment by sjh26** [comment]
This should be good to go.  Included the 5.12.8 update in this since 5.11.2 is no longer in the installer.  James has a PR for 5.15 so we'll merge that after this.

File: Docker/qt5-centos7/Dockerfile [hunk 2]
```diff
@@ -32,13 +35,14 @@ RUN \
...
     libSM \
+    libxkbcommon-x11 \
     libXt-devel \
     libGLU-devel \
     mesa-libOSMesa-devel \
     mesa-libGL-devel \
     mesa-libGLU-devel \
     xcb-util-keysyms \
-    xcb-util-image \
+    xcb-util-image \
```
Comment (jcfr) [inline Docker/qt5-centos7/Dockerfile:45]: Remove extra spaces?

Comment (sjh26) [reply Docker/qt5-centos7/Dockerfile]: Done!

**Pull request merged**